graphein/ml/datasets/torch_geometric_dataset.py

```diff
@@ -225,6 +251,7 @@ def download(self):
                 for pdb in set(self.pdb_codes)
                 if not os.path.exists(Path(self.raw_dir) / f"{pdb}.pdb")
             ]
+        print("downloading uniprotids")
```

Comment:
Using `log` would be better.

Reply:
ohhhhhhhhhhh, too sry for these `print`, forget to remove them, XD. 

I'll remove them today